src/sophys/common/devices/tatu.py

```diff
@@ -286,3 +288,20 @@ class Tatu9403(TatuBase, CRIO_9403):
             "io23": (TatuOutput, "", {"output_number": "23"}),
         }
     )
+
+
+class TatuFlyScan(FlyerInterface):
```

Comment:
This should probably be a subclass of a TATU class, right?

Reply:
Could be, I only left it this way because we established that the Flyscan devices would import the general device, like with the Picolo and Pimega, but as this device is linked to several others, I left it separated.